pkg/systemd/timer-dialog.jsx

```diff
@@ -205,7 +205,7 @@ const CreateTimerDialogBody = ({ owner }) => {
                                        onChange={setDelayNumber} />
                             <FormSelect className="delay-unit"
                                         value={delayUnit}
-                                        onChange={setDelayUnit}
+                                        onChange={(_, val) => setDelayUnit(val)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)